include/wx/gtk/textctrl.h

```diff
@@ -186,6 +186,11 @@ class WXDLLIMPEXP_CORE wxTextCtrl: public wxTextCtrlBase
     void GTKSetWrapMode();
     void GTKSetJustification();
 
+    // get the underlying text buffer for multi-line controls, or null otherwise
+    GtkTextBuffer *GTKGetTextBuffer() { return IsMultiLine() ? m_buffer : nullptr; }
```

Comment:
I think you need to forward-declare it in the beginning of the file, just as it's already done for `GtkTextMark` (which is, ironically, not needed here...).